content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -171,9 +171,7 @@ using the (deprecated) v1alpha2 API instead.
 
 This section outlines the necessary steps to use containerd as CRI runtime.
 
-Use the following commands to install Containerd on your system:
-
-Follow the instructions for [getting started with containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md). Return to this step once you've created a valid configuration file, `config.toml`. 
+To install Containerd on your system, follow the instructions on [getting started with containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md). Return to this step once you've created a valid configuration file, `config.toml`. 
```

Comment:
```suggestion
To install Containerd on your system, follow the instructions on [getting started with containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md). 
Return to this step once you've created a valid configuration file, `config.toml`. 
```

Reply:
@divya-mohan0209 The following changes have been pushed now!